pkg/logged-cluster/options.go

```diff
@@ -4,6 +4,7 @@ package loggedcluster
 type Options struct {
 	EnableLoggingFlag bool
 	LoggingAgent      string
+	EventsLogger      string
```

Comment:
You can add this eventLogger to the options in a separate PR as well :)

Reply:
Well not really as it is needed in several other files with the `getEventsLogger()` function for example :/

Reply:
Even if it's not used

Reply:
Ooooh yeah you're right

Reply:
Smaller PR : https://github.com/giantswarm/logging-operator/pull/268